lib/vision/index.js

```diff
@@ -269,6 +269,15 @@ Vision.prototype.annotate = function(requests, callback) {
  *   //   }
  *   // ]
  * });
+ *
+ * //-
+ * // To handle errors that occurred, check the length of the `errors` array.
```

Comment:
Not trying to be a stickler, but can we elaborate here a little more? Like if we explained how in the event of a malformed image, certain detection types may fail while others will succeed, I think a little more context would be helpful.